packages/util-dynamodb/src/marshall.spec.ts

```diff
@@ -4,62 +4,64 @@ import { marshall } from "./marshall";
 jest.mock("./convertToAttr");
 
 describe("marshall", () => {
-  const mockOutput = { S: "mockOutput" };
-  (convertToAttr as jest.Mock).mockReturnValue({ M: mockOutput });
+  describe("mocked convertToAttr", () => {
```

Comment:
The change is `marshall.spec.ts` is not needed as the unmocked tests are in marshallInput.spec.ts